### PR TITLE
feat: build diary comment create, update, delete api endpoint

### LINKED ISSUE #22: Add Diary Comment API
## 📝 To-do
- [x] add diary comment entity
- [x] build diary comment create API
- [x] build diary comment update API
- [x] build diary comment delete API

### PR DESCRIPTION
**Pull request by thdefn**
## 🔎 PR Description
- Close #22 
> add diary comment create, update, delete api endpoint

## 🔑 Key Changes
- build diary comment create API
- build diary comment update API
- build diary comment delete API
- apply cache to the comment count
- modify DiaryResponse to apply the comment count

## 📢 To Reviewers
- I had a branch conflict on RedisClient, so I modify your code a little @llsrrll96 
- We should discuss about Redis Key rule, because we are both using `opsForValue()` type

### CODE REVIEW

File: petlog-domain/src/main/java/com/ppp/domain/common/constant/CacheValue.java [hunk 1]
```diff
@@ -6,7 +6,9 @@
 @Getter
 @RequiredArgsConstructor
 public enum CacheValue {
-    PET_SPACE_AUTHORITY("petSpaceAuthority");
+    PET_SPACE_AUTHORITY("petSpaceAuthority"),
+    DIARY_COMMENT_COUNT("diaryCommentCount")
+    ;
```
Comment (llsrrll96) [inline petlog-domain/src/main/java/com/ppp/domain/common/constant/CacheValue.java:11]: 큰 건 아니지만 세미클론이 내려갔어욤

File: petlog-domain/src/main/java/com/ppp/domain/diary/DiaryComment.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+public class DiaryComment extends BaseTimeEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(columnDefinition = "blob", nullable = false)
+    private String content;
+
+    @Column(name = "tagged_users", columnDefinition = "json")
+    @JdbcTypeCode(SqlTypes.JSON)
```
Comment (llsrrll96) [inline petlog-domain/src/main/java/com/ppp/domain/diary/DiaryComment.java:30]: JSON 을 쓰는 군요! 👍

**Review by llsrrll96 — APPROVED** [review]
very good

File: petlog-domain/src/main/java/com/ppp/domain/common/constant/CacheValue.java (hunk 1 above)
Comment (thdefn) [reply petlog-domain/src/main/java/com/ppp/domain/common/constant/CacheValue.java:11]: 오알려주셔서감사합니당

File: petlog-domain/src/main/java/com/ppp/domain/diary/DiaryComment.java (hunk 2 above)
Comment (thdefn) [reply petlog-domain/src/main/java/com/ppp/domain/diary/DiaryComment.java:30]: 넵 ! 태깅이 리스트라서 타입을 고민하다가 json으로 채택했어요 🚀

**Pull request merged**